modules/administration-guide/partials/proc_restoring-a-che-instance-from-a-backup-by-using-the-checlusterrestore-object.adoc

```diff
@@ -0,0 +1,64 @@
+[id="restoring-a-{prod-id-short}-instance-from-a-backup-by-using-the-checlusterrestore-object_{context}"]
+= Restoring a {prod-short} instance from a backup by using the CheClusterRestore custom object
+
+You can restore a {prod-short} instance from a backup snapshot by using custom resources. A new `CheClusterRestore` custom object is required every time you restore a {prod-short} instance from a backup snapshot; that is, editing an already consumed `CheClusterRestore` custom object, during or after backing up, has no effect.
+
+include::snip_warning-backup-snapshots-are-bound-to-specific-cluster.adoc[]
+
+IMPORTANT: The `CheClusterRestore` custom object is unusable for recovering a {prod-short} instance of an earlier version of {prod-short}. Use `{prod-cli}` to recover a {prod-short} instance of an earlier version of {prod-short}; see xref:restoring-a-che-instance-from-a-backup-by-using-prod-cli_{context}[].
```

Comment:
nitpick: this is more my fault, but it is still possible if admin manually deletes current and deploy required version of Che and the apply the procedure.

Reply:
@mmorhun I can fix it. Could you clarify what you want me to change or add in this IMPORTANT admonition?

Reply:
I don't think we should hightlight that, but just let users know. Preferred way is to use chectl, but just in case, if a user doesn't want to use it for some reason, we can tell about a long way.

Reply:
@mmorhun, no worries, I've changed it to:
```
IMPORTANT: Do not use the `CheClusterRestore` custom object 
to recover a {prod-short} instance of an earlier version of {prod-short}! 
Use only `{prod-cli}` to recover a {prod-short} instance of 
an earlier version of {prod-short}; 
see xref:restoring-a-che-instance-from-a-backup-by-using-prod-cli_{context}[]!
```